docker-compose.yaml

```diff
@@ -29,6 +29,10 @@ services:
         - TCF_DEBUG_BUILD=${TCF_DEBUG_BUILD:-}
         - DISPLAY=${DISPLAY:-}
         - XAUTHORITY=~/.Xauthority
+    ports:
+      - "1947:1947"
+      - "5000:5000"
```

Comment:
What is port 5000 needed for?
I see port 5555 used for ZMQ, but no usage of port 5000 anywhere.

Reply:
Remove this line. TCP Port is not used by Avalon or ZMQ.

Reply:
The 5000 port is used by the Colchain client flask server. File location -examples/apps/cold_chain_eval/client/ColdChainAPI.py

Reply:
OK. Add a 1 line comment for what ports 5000 and 555 are used for.